kusto-language-server/client/src/extension.ts

```diff
@@ -105,6 +105,16 @@ commands.registerCommand('kuskus.loadSymbols', async () => {
 			return;
 		}
 
+		const authId = await window.showInputBox({
+			ignoreFocusOut: true,
+			placeHolder: 'common',
+			prompt: 'Authority/Tenant ID'
+		});
+		if (!authId) {
```

Comment:
If the user just immediately hits enter, does `window.showInputBox` return the placeholder value of "common" or will it return null / empty string? Aka if the user just hits enter, do they get this error message or nah?

Reply:
i'll test this and try to enable that functionality if it doesn't do that.